tests/v2/actions/charts/stackstate.go

```diff
@@ -0,0 +1,287 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+
+	catalogv1 "github.com/rancher/rancher/pkg/apis/catalog.cattle.io/v1"
+	kubenamespaces "github.com/rancher/rancher/tests/v2/actions/kubeapi/namespaces"
+	"github.com/rancher/rancher/tests/v2/actions/namespaces"
+	"github.com/rancher/rancher/tests/v2/actions/observability"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/defaults"
+	"github.com/rancher/shepherd/pkg/api/steve/catalog/types"
+	"github.com/rancher/shepherd/pkg/wait"
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	// Public constants
+	StackstateExtensionNamespace = "cattle-ui-plugin-system"
+	StackstateExtensionsName     = "observability"
+	UIPluginName                 = "rancher-ui-plugins"
+	StackstateK8sAgent           = "stackstate-k8s-agent"
+	StackstateNamespace          = "stackstate"
```

Comment:
Should you change this to "suse-observability" to match the install documentation? The namespace really doesn't matter, but it might make it easier for others to follow along in the future.

Reply:
this is for stackstate agent chart.. docs have the namespace as `stackstate` in the chart installation https://docs.stackstate.com/5.1/setup/agent/kubernetes-openshift#helm-chart-values 

```helm upgrade --install \
  --namespace stackstate \
  --create-namespace \
```